### PR TITLE
Add content for the Fourth year undergraduate courses

### PR DESCRIPTION
**Pull request by Mutti499**
Questions to solve:
- can prerequisites only refer to courses or can they include the consent of the lecturer, the requirement to be a senior or the requirements written in the course description (such as cmpe434)?
- cmpe462 has never been given in the fall period except this year. Still, would it be appropriate to write fall in the content
- It does not appear anywhere except the faculty page for cmpe470, I could not see the course in the past years. Where can I find information about the course?
- will the course outcome be written for cmpe492
- Shall we start uploading class and instructor photos in our updates?

### CODE REVIEW

**Comment by doganulus** [comment]
> can prerequisites only refer to courses or can they include the consent of the lecturer, the requirement to be a senior or the requirements written in the course description (such as cmpe434)?

CMPE4XX courses do not require the SENIOR status except CMPE491/492. CMPE434 does not have a prerequisite for example. So we must write None.

> cmpe462 has never been given in the fall period except this year. Still, would it be appropriate to write fall in the dopnemine

Let's write the latest condition.

> It does not appear anywhere except the faculty page for cmpe470, I could not see the course in the past years. Where can I find information about the course?

Let me check it. But we can skip it for now.

> will the course outcome be written for cmpe492

Yes. But we can skip it if we are not sure. 

> Shall we start uploading class and instructor photos in our updates?

Yes. We need to do that at some point. But let's keep it as a separate PR.

File: content/courses/cmpe470.en.md [hunk 1]
```diff
@@ -7,9 +7,40 @@ aliases:
...
+
+<!-- prettier-ignore-start -->
+{{< table class="table-hover table-sm" >}}
+|||
+| :-- | :-- |
+| Faculty | Faculty of Engineering |
+| Course Code | CMPE470 |
+| Course Title | Computer Performance Evaluation |
+| Language of Instruction | English |
+| Course Semester | -- |
```
Comment (Mutti499) [inline content/courses/cmpe470.en.md:20]: Not written in the [reference](https://www.bogazici.edu.tr/tr_TR/Content/Akademik/Lisans_Katalogu/Muhendislik_Fakultesi/Bilgisayar_Muhendisligi_Bolumu)

Comment (Mutti499) [reply content/courses/cmpe470.en.md:20]: I actually did checked the [reference page](https://www.bogazici.edu.tr/tr_TR/Content/Akademik/Lisans_Katalogu/Muhendislik_Fakultesi/Bilgisayar_Muhendisligi_Bolumu) you shared and put that info already. However that info is not helping me to fill spaces like course semester or outcome of the course

Comment (Mutti499) [reply content/courses/cmpe470.en.md:20]: According to the latest information I got from my friends, the 470 course has not been open for years. I'm not sure the page is supposed to exist.

**Comment by Mutti499** [comment]
@doganulus ready for review!

**Review by doganulus — APPROVED** [review]
Unify warning shortcodes using Hugo's i18n feature.

**Pull request merged**